wbooks/src/components/home/Home.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import Header from '../header/Header';
+import BookCard from '../book-card/BookCard';
+
+import { styles } from './Home.styles'
+
+class Home extends Component {
+  constructor() {
+    super();
+    this.state = {
+      items: [],
+      filterType: null,
+      filterData: ''
+    }
+  }
+
+  componentWillMount() {
+    let data = require('../../data/books.json');
+    this.setState({
+      items: data
+    })
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    this.setState({
+      filterType: this.type.value,
+      filterData: this.data.value,
+    });
+  }
+
+  render() {
+    const items = this.state.items;
```

Comment:
you shouldnt do anything here, just the return. So if you want to get a live reload, use the input onChange function and put this comparations there 😄